### DISCUSSION TITLE
AI培菌師

### DISCUSSION THREAD

**Discussion by number32984** [1101-GS3073-C/Python-Community-GS3073-C#613] [General]
我都沒想到AI居然可以用在這種地方，厲害了我的AI
https://vip.udn.com/vip/story/121938/6889428

**Comment by larry40305** [comment]
以後AI會取代越來越多的人力，也能取代更多比較專業的人力。

**Comment by awkward-willy** [comment]
透過 AI 科技與感測器等地相互配合，能讓農業所需的人數大幅減少，同時增進品質與產量，甚至減少因不可控的天災等因素影響，可見科技農業是未來發展的一大主軸之一。

**Comment by manipulateed** [comment]
AI可以弭補人無法隨時監測的缺點，而適時給予作物調整，能讓生產更加便利，讓勞力更加得到釋放，進而給予作物最好的生長環境。

**Comment by boson13579** [comment]
藉由AI調控香檳茸生長的溫度、濕度相對於人工會更為精確，同時也省下了人力成本，並提高生產成功率。
